truckdevil/j1939/j1939.py

```diff
@@ -171,9 +172,15 @@ def print_messages(self, abstract_tpm=True, read_time=None, num_messages=None, v
                 continue
             # Print/log the message
             if not verbose:
-                print(j1939_message)
-                if log_to_file:
-                    log_file.write(str(j1939_message) + '\n')
+                if not candump:
+                    print(j1939_message)
+                    if log_to_file:
+                        log_file.write(str(j1939_message) + '\n')
+                else:
+                    if j1939_message.total_bytes < 9: 
+                        print(self.get_candump(j1939_message))
```

Comment:
There is an uncaught exception on get_candump here. Everything within the else statement should be wrapped in a a try block.

Reply:
I'll take a look at it and fix. Thanks for pointing that out.